Corona-Warn-App/src/main/java/de/rki/coronawarnapp/datadonation/analytics/storage/AnalyticsSettings.kt

```diff
@@ -89,12 +89,35 @@ class AnalyticsSettings @Inject constructor(
         defaultValue = false
     )
 
+    val testScannedAfterConsent = prefs.createFlowPreference(
+        key = PREFS_KEY_TEST_SCANNED_AFTER_CONSENT,
+        reader = { key ->
```

Comment:
I think for primitives `FlowPreference` has direct support, you can just specify the default value.